build

```diff
@@ -1089,6 +1101,9 @@ validate_buildroot "$BUILD_ROOT"
 # done option parsing
 BUILD_OPTIONS_PARSED=true
 
+test -n "$CCACHE" && CCACHE="$BUILD_ROOT/.build.oldpackages/_ccache.tar.gz"
```

Comment:
Why is this ".build.oldpackages" suddenly the default for --ccache?
Why use ".build.oldpackages" at all?

Reply:
because the ccache tar is from preivously built package.

Reply:
No it's not, it's from the --pkg-ccache argument.